go.mod

```diff
@@ -18,22 +18,22 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
-	github.com/submariner-io/admiral v0.15.2
-	github.com/submariner-io/cloud-prepare v0.15.2
-	github.com/submariner-io/submariner v0.15.2
-	github.com/submariner-io/submariner-operator v0.15.2
+	github.com/submariner-io/admiral v0.15.3
+	github.com/submariner-io/cloud-prepare v0.15.3
+	github.com/submariner-io/submariner v0.15.3
+	github.com/submariner-io/submariner-operator v0.15.3
 	golang.org/x/oauth2 v0.13.0
 	google.golang.org/api v0.148.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.26.7
-	k8s.io/apiextensions-apiserver v0.26.7
-	k8s.io/apimachinery v0.26.7
-	k8s.io/client-go v0.26.7
-	k8s.io/code-generator v0.26.7
-	k8s.io/component-base v0.26.7
+	k8s.io/api v0.26.10
+	k8s.io/apiextensions-apiserver v0.26.10
+	k8s.io/apimachinery v0.26.10
+	k8s.io/client-go v0.26.10
+	k8s.io/code-generator v0.26.10
+	k8s.io/component-base v0.26.10
 	k8s.io/klog/v2 v2.100.1
-	open-cluster-management.io/addon-framework v0.6.1
-	open-cluster-management.io/api v0.10.1-0.20230216050312-b716d4a5c514
+	open-cluster-management.io/addon-framework v0.8.0
+	open-cluster-management.io/api v0.12.0
```

Comment:
The integration tests panic b/c of one of these bumps. Why were these bumped? Is 2.8 compatible with these versions?

Reply:
`go get` bumped them, I’ve reverted that.